libcudacxx/include/cuda/std/__cccl/builtin.h

```diff
@@ -127,6 +127,18 @@
 #  undef _CCCL_BUILTIN_BIT_CAST
 #endif // clang < 10 || nvcc < 11.7
 
+#if _CCCL_HAS_BUILTIN(__builtin_COLUMN) || defined(_CCCL_COMPILER_MSVC) && _CCCL_MSVC_VERSION >= 1927
+#  define _CCCL_BUILTIN_COLUMN() __builtin_COLUMN()
+#else // ^^^ _CCCL_HAS_BUILTIN(__builtin_COLUMN) ^^^ / vvv !_CCCL_HAS_BUILTIN(__builtin_COLUMN) vvv
+#  define _CCCL_BUILTIN_COLUMN() 0
+#endif // !_CCCL_HAS_BUILTIN(__builtin_COLUMN)
+
+// NVCC below 11.3 cannot handle __builtin_COLUMN
+#if defined(_CCCL_CUDACC_BELOW_11_3)
+#  undef __builtin_COLUMN
+#  define __builtin_COLUMN() 0
+#endif // _CCCL_CUDACC_BELOW_11_3
```

Comment:
This is a convention we started because it becomes really difficult to track what is enabled and what is not at times.

So we settled on a convention that we have one condition that enables something and one condition that explicitly disables something if there is a compiler bug

Reply:
Makes sense, thanks.